src/app/models/submission.server.model.ts

```diff
@@ -41,6 +44,11 @@ const SubmissionSchema = new Schema<ISubmissionSchema>(
         },
       ],
     },
+    submissionType: {
+      type: String,
+      enum: Object.values(SubmissionType),
+      required: true,
+    },
```

Comment:
we missed out on a key in the Submissions collection?

Reply:
This is a discriminator key, I'm just making it more obvious